website/src/utils/ownership-statuses.js

```diff
@@ -0,0 +1,12 @@
+export const CLAIM_PROFILE = 'CLAIM_PROFILE'
+export const EDIT_PROFILE = 'EDIT_PROFILE'
+export const NOT_CONNECTED_PROFILE = 'NOT_CONNECTED_PROFILE'
+export const LOADING_PROFILE = 'LOADING_PROFILE'
+export const REQUEST_EDIT_PROFILE = 'REQUEST_EDIT_PROFILE'
+
+export const OWNERSHIP_STATUSES = new Map([
+  ['CLAIM_PROFILE', 'Claim profile'],
+  ['EDIT_PROFILE', 'Edit profile'],
+  ['NOT_CONNECTED_PROFILE', 'Claim Profile'],
```

Comment:
```suggestion
  ['NOT_CONNECTED_PROFILE', 'Claim profile'],
```

Adjusting text case to make sure they are the same style in all of the cases